app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileDestination.kt

```diff
@@ -214,6 +218,73 @@ private fun ProfileScreen(
   }
 }
 
+@Composable
+private fun ColumnScope.ProfileRowsWithPlaceholders(
+  profileUiState: ProfileUiState,
+  showMyInfo: () -> Unit,
+  showSettings: () -> Unit,
+  showAboutApp: () -> Unit,
+  navigateToEurobonus: () -> Unit,
+  navigateToTravelCertificate: () -> Unit,
+) {
+  AnimatedContent(
+    targetState = profileUiState.isLoading,
+    transitionSpec = { fadeIn() togetherWith fadeOut() },
+  ) { isLoading ->
+    Column(Modifier.fillMaxSize()) {
+      when (isLoading) {
+        true -> {
+          ProfileItemRowsPlaceholders()
+        }
+
+        false -> {
+          ProfileItemRows(
+            profileUiState = profileUiState,
+            showMyInfo = showMyInfo,
+            showSettings = showSettings,
+            showAboutApp = showAboutApp,
+            navigateToEurobonus = navigateToEurobonus,
+            navigateToTravelCertificate = navigateToTravelCertificate,
+          )
+        }
+      }
+    }
+  }
+}
+
+@Composable
+private fun ColumnScope.ProfileItemRowsPlaceholders() {
+  ProfileRowPlaceholder(stringResource(R.string.PROFILE_MY_INFO_ROW_TITLE))
+  ProfileRowPlaceholder(stringResource(R.string.PROFILE_ROW_TRAVEL_CERTIFICATE))
+  ProfileRowPlaceholder(stringResource(R.string.PROFILE_ABOUT_ROW))
+  ProfileRowPlaceholder(stringResource(R.string.profile_appSettingsSection_row_headline))
+}
+
+@Composable
+private fun ProfileRowPlaceholder(title: String) {
+  Row(
+    verticalAlignment = Alignment.CenterVertically,
+    modifier = Modifier
+      .fillMaxWidth()
+      .padding(16.dp),
+  ) {
+    Icon(
+      imageVector = Icons.Hedvig.Info,
+      contentDescription = null,
+      modifier = Modifier
+        .size(24.dp)
+        .placeholder(true, highlight = PlaceholderHighlight.fade()),
+    )
+    Spacer(Modifier.width(16.dp))
+    Text(
+      text = title,
+      style = MaterialTheme.typography.bodyLarge,
+      modifier = Modifier
+        .placeholder(true, highlight = PlaceholderHighlight.shimmer()),
+    )
+  }
+}
+
 @Composable
 private fun ColumnScope.ProfileItemRows(
```

Comment:
```suggestion

@Suppress("UnusedReceiverParameter")
@Composable
private fun ColumnScope.ProfileItemRows(
```
Here I would suggest this, since we do in fact want to know that we are inside a ColumnScope, since we emit mutltiple composables without explicitly wrapping them in some container like Column.